src/azure/cli/_help.py

```diff
@@ -212,7 +213,14 @@ def __init__(self, delimiters):
         self.examples = ''
 
     def load(self, options):
-        self.short_summary = getattr(options, 'description', None)
+        description = getattr(options, 'description', None)
+        try:
+            self.short_summary = description[:description.index('.')]
+            self.long_summary = description[description.index('.') + 1:].lstrip()
+        except (ValueError, AttributeError):
+            self.short_summary = description
+            self.long_summary = None
```

Comment:
Good point. I had it the other way around originally but that caused the help tests to fail, so I changed it to conform to the tests. Unit test win. 

---
In reply to: [76083810](https://github.com/Azure/azure-cli/pull/736#discussion_r76083810) [](ancestors = 76083810)